session-vbox/TODO

```diff
@@ -0,0 +1,11 @@
+== TODO
+
+ * Need to set bidirectional shared clipboard when creating disk image itself.
```

Comment:
`vboxmanage modifyvm --clipboard bidirectional` can be used while creating the disk image for enabling bidirectional shared clipboard.